resources/axon.rb

```diff
@@ -58,9 +58,30 @@
 
   # Set platform specific variables
   case node['platform']
-  when 'centos', 'redhat', 'suse', 'oraclelinux', 'oracle', 'amazon'
+  when 'centos', 'redhat', 'suse', 'oraclelinux', 'amazon'
     ext = '.rpm'
     eg_service_name = 'tw-eg-service'
+  when 'oracle'
```

Comment:
Should this be an if embedded in the earlier clause so we don't have to repeat lines 62 and 63?

Reply:
Done